### DISCUSSION TITLE
Pydantic Survey - invitation to participate

### DISCUSSION THREAD

**Discussion by samuelcolvin** [pydantic/pydantic#4692] [Pydantic V2]
As explained [on twitter](https://twitter.com/samuel_colvin/status/1585958365380825088) I'm conducting a survey of how people use Pydantic to inform development of [Pydantic V2](https://pydantic-docs.helpmanual.io/blog/pydantic-v2/).

The survey is open to sponsors of pydantic and anyone who has contributed to pydantic.

However I have no way of getting in touch with the following users to send them the link:

@zachkirsch @Natim @mfulgo @Bobronium @daviskirk @hmvp @therefromhere @DetachHead @paxcodes @art049 @graingert @LasseGravesen @oldPadavan @commonism @JeanArhancet @garyd203 @christianbundy @maximberg @Mazyod @Hultner @tayoogunbiyi @masalim2 @aimestereo @mdavis-xyz @Atheuz @gsakkis @leosussan @matin @YannLuo @mgu @retnikt

If you're one of those users (or indeed anyone else who's contributed to pydantic), please get in touch with me via either [`@samuel_colvin` :bird:](https://twitter.com/samuel_colvin) or email `s _at_ muelcolvin.com`, and I'll send you a link.

**Comment by Hultner** [comment]
I sent you a dm on Twitter

**Comment by garyd203** [comment]
I'll send you a DM on twitter too 👍

**Comment by samuelcolvin** [comment]
@abegerho thanks so much for sponsoring me. I don't have a way to contact you without emailing all the people on that tier except a message like this.

See the above message, if you'd like to take the survey please message me and I'll send you the link.

**Reply by abegerho** [reply]
I can be reached at: abhishek@314ecorp.com. Unfortunately, I am not a programmer myself. We use FastAPI and I want to send some token appreciation to you as well as I was told that FastAPI builds on top of Pydantic. We are new to the software world. If our software sells well, I hope to be at a higher tier. Thank you for your contributions to the open source community. Gratitude!

**Comment by christianbundy** [comment]
Sent you an email btw.

**Reply by samuelcolvin** [reply]
Hi @christianbundy, yes I replied, let me know if you didn't get the email and I'll try again.

**Reply by christianbundy** [reply]
Ah, went straight to spam -- found it now, thanks!

**Comment by DetachHead** [comment]
sent you an email

**Comment by abegerho** [comment]
Just so you have my regular email.

**Reply by samuelcolvin** [reply]
Hi @abegerho unfortunately I still can't see your email address, it's obfuscated by github. You'll have to contact me directly.
